client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/test/index.js

```diff
@@ -38,7 +38,7 @@ describe( 'Carrier Accounts', () => {
 	it('should render a list of provided accounts', () => {
 		const wrapper = mount(
 			<Wrapper>
-				<CarrierAccounts carriers={[
+				<CarrierAccounts accounts={[
```

Comment:
Just renames on the prop

Reply:
:information_source: Missing spaces inside braces.